client/thin-replica-client/CMakeLists.txt

```diff
@@ -30,6 +30,7 @@ target_link_libraries(thin_replica_client_lib
   util
   cryptopp
   thin-replica-proto
+  logging
```

Comment:
no need, it's implicitly linked to all targets

Reply:
Thanks @toly-kournik! Will remove it.